tools-v2/pkg/cli/command/curvebs/check/server/server.go

```diff
@@ -0,0 +1,250 @@
+/*
+ *  Copyright (c) 2022 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: CurveCli
+ * Created Date: 2023-05-07
+ * Author: pengpengSir
+ */
+package server
+
+import (
+	"context"
+	"fmt"
+
+	cmderror "github.com/opencurve/curve/tools-v2/internal/error"
+	cobrautil "github.com/opencurve/curve/tools-v2/internal/utils"
+	basecmd "github.com/opencurve/curve/tools-v2/pkg/cli/command"
+
+	copysetbs "github.com/opencurve/curve/tools-v2/pkg/cli/command/curvebs/status/copyset"
+	"github.com/opencurve/curve/tools-v2/pkg/config"
+	"github.com/opencurve/curve/tools-v2/pkg/output"
+	"github.com/opencurve/curve/tools-v2/proto/proto/copyset"
+	"github.com/opencurve/curve/tools-v2/proto/proto/topology"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+)
+
+const (
+	serverExample = `$ curve bs check server`
+)
+
+type CheckServerRpc struct {
+	Info           *basecmd.Rpc
+	Request        *topology.ListChunkServerRequest
```

Comment:
add listchunkserver in [list](tools-v2/pkg/cli/command/curvebs/list/chunkserver/).
However, [other pr](https://github.com/opencurve/curve/pull/2386) is already doing this part of the content, and I have explained the relevant amendments.

Reply:
Or you can communicate with him and ask him to reserve a corresponding interface so that you can continue.